node_exporter.go

```diff
@@ -168,7 +173,18 @@ func main() {
 	})
 
 	log.Infoln("Listening on", *listenAddress)
-	if err := http.ListenAndServe(*listenAddress, nil); err != nil {
+
+	server := &http.Server{Addr: *listenAddress, Handler: nil}
```

Comment:
nil is the default

Reply:
Is this nil being default for the handler?

Reply:
Yes

Reply:
```suggestion
	server := &http.Server{Addr: *listenAddress}
```